components/raftstore/src/store/fsm/store.rs

```diff
@@ -1081,7 +1082,7 @@ pub struct RaftPollerBuilder<EK: KvEngine, ER: RaftEngine, T> {
     split_check_scheduler: Scheduler<SplitCheckTask>,
     cleanup_scheduler: Scheduler<CleanupTask>,
     raftlog_gc_scheduler: Scheduler<RaftlogGcTask>,
-    raftlog_fetch_scheduler: Scheduler<RaftlogFetchTask>,
+    raftlog_fetch_scheduler: Scheduler<ReadTask<EK>>,
```

Comment:
Field name not updated.

Reply:
I will keep it as `raftlog_fetch_scheduler` in v1 first. I can change it in a separated PR if needed.